apps/vscode/justfile

```diff
@@ -0,0 +1,8 @@
+install:
```

Comment:
Taking example on Biome, we've started using the `just` command runner in Ark and Air. I've started one here to easily update dev Quarto in Code and Positron.